provisioning/roles/nodejs/templates/package.json.webpack.j2

```diff
@@ -9,26 +9,25 @@
     "build": "NODE_ENV=production webpack"
   },
   "devDependencies": {
-    "@babel/core": "^7.1.2",
-    "@babel/preset-env": "^7.1.0",
-    "autoprefixer": "^9.1.5",
-    "babel-loader": "^8.0.4",
-    "css-loader": "^1.0.0",
-    "cssnano": "^4.1.4",
-    "file-loader": "^2.0.0",
-    "mini-css-extract-plugin": "^0.4.0",
-    "node-sass": "^4.8.3",
-    "postcss": "^7.0.5",
-    "postcss-cli": "^6.0.0",
+    "@babel/core": "^7.4.5",
+    "@babel/preset-env": "^7.4.5",
+    "autoprefixer": "^9.6.0",
+    "babel-loader": "^8.0.6",
+    "css-loader": "^2.1.1",
```

Comment:
I see a major bump here, we should ensure our configuration/usage is still adapted.

Reply:
Yeah, there is a link to the changes in my first comment. They changed the way imports are handled, but in our case `sass-loader` already handles imports before `css-loader` (as far as I understand).

However I could have missed something, this is worth another look.

Reply:
Oh sorry, I missed your initial statement, it indeed looks good. I'll try to setup it once from scratch just to be sure.

Reply:
I tried it on a project and it worked just fine, I think we can merge this.

Reply:
Sure then, go ahead!